### DISCUSSION TITLE
Always an update available

### DISCUSSION THREAD

**Discussion by yovanoc** [uNetworking/uWebSockets.js#162] [Q&A] [answered]
Why yarn is always telling me that ?
<img width="769" alt="Capture d’écran 2019-06-11 à 09 54 29" src="https://user-images.githubusercontent.com/7302317/59253648-fcd96280-8c2e-11e9-8de6-7a9ecc988948.png">

**Comment by ghost** [comment]
If you have issues with yarn then ask them. I have no idea what yarn does

**Comment by yovanoc** [comment]
There is no issues with yarn, I'm using it every day in a lot of projects and he never tell me that there is an "exotic" update available when i'm at the latest version of the packages, so i'm asking here

**Comment by ghost** [comment]
I don't know

**Comment by yovanoc** [comment]
Ok it's not really bothering but i'm sure there is a problem somewhere, so no worries

**Comment by coder456** [comment, marked as answer]
@yovanoc you try google this at all?
https://stackoverflow.com/questions/49259346/yarn-upgrade-interactive-always-to-exotic

**Comment by yovanoc** [comment]
Oh so yes this package is not on npm, right, mb
